modules/quickstart-public-cloud/pages/installation.adoc

```diff
@@ -18,10 +18,10 @@ Follow the prompts, and wait for the setup to finish.
 
 For detailed instructions on setting up {productname} with {yast}, see xref:installation:server-setup.adoc[].
 
-Register your instance with {scc}, using this command:
+Register your instance with {scc}, using this command, specifying the registration number and the email address associated with it:
 
 ----
-sudo SUSEConnect -r <Registration_Number>
+sudo SUSEConnect -r <Registration_Number> -e <registered_email_address>
```

Comment:
the sudo way is not wrong but we usually say run this command as root. We probably should have this in the intro section of a book.